controllers/api/server.go

```diff
@@ -71,6 +71,8 @@ func (as *Server) registerRoutes() {
 	router.HandleFunc("/import/group", as.ImportGroup)
 	router.HandleFunc("/import/email", as.ImportEmail)
 	router.HandleFunc("/import/site", as.ImportSite)
+  router.HandleFunc("/webhooks/", mid.Use(as.Webhooks, mid.RequirePermission(models.PermissionModifySystem)))
+  router.HandleFunc("/webhooks/{id:[0-9]+}/ping", mid.Use(as.PingWebhook, mid.RequirePermission(models.PermissionModifySystem)))
```

Comment:
Let's change this to `validate` instead of `ping`. This will match the pattern that we're doing with the IMAP additions [here](https://github.com/gophish/gophish/pull/1612/files#diff-fab0ab83fa5d22c823eaf5fea7c04cabR52)

Reply:
Ok. But using verbs in the URL in REST API  is discouraged. It's recommended to use nouns. It'd be "/validation"

Reply:
I'd prefer to leave it `validate`. I understand that REST API semantics prefer for everything in the URL to be a resource, but to me `validate` reads cleaner in that we're validating a resource, not referring to a new validation.

I appreciate the suggestion, though! Keep up the great work.